src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +47,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        try (Session session = sessionFactory.openSession()) {
+            Query<Movie> getMovieQuery = session.createQuery("from Movie", Movie.class);
+            return getMovieQuery.list();
+        } catch (Exception e) {
+            throw new DataProcessingException("Can't get all movie", e);
```

Comment:
The error message in the `DataProcessingException` should be pluralized to 'Can't get all movies' for consistency and clarity.